datadog-agent-nvml.yaml

```diff
@@ -0,0 +1,115 @@
+package:
+  name: datadog-agent-nvml
+  version: 1.0.9
+  epoch: 0
+  description: "Checks NVIDIA Management Library (NVML) exposed metrics through the Datadog Agent and can correlate them with the exposed Kubernetes devices"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
```

Comment:
Last thing - we should likely set `no-provides` and `no-depends` as we're creating a virtual env for this below. We don't want anything to install this if something happens to be vendored that another package depends on, and we don't want this to pull in other packages if deps are already vendored (if we need anything with packages built within virtual envs, we should explicitly set it as a runtime dep)